nixos/modules/services/audio/jmusicbot.nix

```diff
@@ -0,0 +1,42 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+let
+  cfg = config.services.jmusicbot;
+in
+{
+  options = {
+    services.jmusicbot = {
+      enable = mkEnableOption "jmusicbot, a Discord music bot that's easy to set up and run yourself!";
+
+      serversettings = mkOption {
```

Comment:
Opinion which should you should feel free to ignore if so inclined: you might want to call this `stateDir` or `configDir` to better match other modules.